lib/tower_cli/resources/job_template.py

```diff
@@ -58,8 +58,7 @@ class Resource(models.Resource):
     job_tags = models.Field(required=False, display=False)
     skip_tags = models.Field(required=False, display=False)
     extra_vars = models.Field(required=False, display=False)
-    become_enabled = models.Field(type=bool, required=False, display=False,
-                                  show_default=True, default=False)
```

Comment:
Yes, this has to do with how the CLI interacts with the API, and I should explain this. Fields that are not required will generally (reasonably) not use a default. These fields will not even be included when it does a POST. In the JSON data it's not true or false, it's just not there. Not including a field means that we're relying on whatever kind of default the Tower server has set.